### PR TITLE
fix: strip ansi markup for error message

### PR DESCRIPTION
**Pull request by sipayRT**

### CODE REVIEW

File: test/func/tests/common/test-results-appearance.testplane.js
Comment (shadowusr) [inline test/func/tests/common/test-results-appearance.testplane.js:1]: We don't need those awaits as far as I can tell... But of course they won't break nothing.

File: test/func/fixtures/testplane/failed-describe.testplane.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ describe('failed describe', function() {
         throw new Error('Some error');
     });
 
+    it('failed test with ansi markup', async () => {
+        await expect({a: {b: 'c'}}).toMatchObject({c: {b: 'a'}});
```
Comment (KuznetsovRoman) [inline test/func/fixtures/testplane/failed-describe.testplane.js:33]: What kind of test is it?

Comment (KuznetsovRoman) [inline test/func/fixtures/testplane/failed-describe.testplane.js:33]: Do we need await here?

**Review by shadowusr — APPROVED** [review]

**Review by KuznetsovRoman — APPROVED** [review]

File: test/func/fixtures/testplane/failed-describe.testplane.js (hunk 1 above)
Comment (sipayRT) [reply test/func/fixtures/testplane/failed-describe.testplane.js:33]: actually no, but I added it for consistency

**Pull request merged**